tilelang/backend/target.py

```
@@ -0,0 +1,75 @@
+def register_lazy_target_detector(name: str, import_path: str) -> None:
+    _LAZY_TARGET_DETECTORS[name] = import_path
+
+
+def _ensure_target_detectors_loaded() -> list[str]:
+    errors: list[str] = []
+    for name, import_path in tuple(_LAZY_TARGET_DETECTORS.items()):
+        if name in _LOADED_TARGET_DETECTORS:
+            continue
+        try:
+            import_module(import_path)
+        except Exception as err:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)